src/components/Tabs/TabsPane.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import './TabsPane.scss';
+
+const propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+    isSelected: PropTypes.bool,
+    relatedTabId: PropTypes.string,
+};
+
+const defaultProps = {
+    children: null,
+    className: null,
+    isSelected: false,
+    relatedTabId: null,
+};
+
+class TabsPane extends Component {
+    state = {
+        isDeselecting: false,
+        isSelecting: false,
+    };
+
+    componentWillReceiveProps(nextProps) {
```

Comment:
`componentWillReceiveProps` is a little verbose... could this work as a more concise option:
```
componentWillReceiveProps(nextProps) {
    this.setState({
        isDeselecting: this.props.isSelected && !nextProps.isSelected,
        isSelecting: !this.props.isSelected && nextProps.isSelected,
    });
}
```

Reply:
oh, yes, good point!

Reply:
Fixed